tasks/build.py

```diff
@@ -760,3 +760,65 @@ def check_build_permission(pr, event_info):
     else:
         log(f"{fn}(): GH account '{build_labeler}' is authorized to build")
         return True
+
+
+def request_bot_build_issue_comments(repo_name, pr_number):
+    status_table = {'arch': [], 'date': [], 'status': [], 'url': [], 'result': []}
```

Comment:
Wonder a bit if it would be better to let `status_table` be a list of dictionaries instead of a dictionary of lists where we assume that lists are always identically formed (same length, elements at same index belong to each other).

Reply:
I'm open to that. Because it actually takes some effort to put it in a dictionary with lists.

Reply:
Ack. We can think about that later in case it becomes necessary.